n3fit/src/n3fit/layers/CombineCfac.py

```diff
@@ -123,7 +123,7 @@ def call(self, inputs, linear_values):
                 dtype=np.float32,
             )
             linear = self._compute_linear(linear_values)
-            inputs = tf.cast(inputs, tf.float64)
+            inputs = tf.cast(inputs, tf.float32)
 
```

Comment:
```suggestion
linear = tf.cast(linear, tf.float32)
```
This change fixes the error for me.  @FrancescoMerlotti  Do you think that this might be okay